app/components/tracks/TracksController.js

```diff
@@ -31,6 +33,57 @@ tracksModule.controller('TracksController',
                 }, function (error) {
                     console.error(error);
                 });
+
+            ApiJsonFactory.getJson('sessions')
+                .then(function (response) {
+                    tc.Sessions = response.data.sessions;
+                    $sessionStorage.sessions = tc.Sessions;
+                }, function (error) {
+                    console.error(error);
```

Comment:
I think you can continue using console. All modern browsers support console.

On 26 September 2015 at 07:23, Saptak Sengupta <notifications@github.com>
wrote:

> In app/components/tracks/TracksController.js
> <https://github.com/fossasia/open-event-webapp/pull/75#discussion_r40488845>
> :
>
> > @@ -31,6 +33,57 @@ tracksModule.controller('TracksController',
> >                  }, function (error) {
> >                      console.error(error);
> >                  });
> > +
> > +            ApiJsonFactory.getJson('sessions')
> > +                .then(function (response) {
> > +                    tc.Sessions = response.data.sessions;
> > +                    $sessionStorage.sessions = tc.Sessions;
> > +                }, function (error) {
> > +                    console.error(error);
>
> As far as I saw, all the other controller codes also used console. So, I
> stuck with console . If this needs to be fixed then I guess it needs to
> be fixed for all. Then it deserves to be an issue.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fossasia/open-event-webapp/pull/75/files#r40488845>.
>